default/stringtables/en.txt

```diff
@@ -339,11 +339,14 @@ PROJECT_BT_STOCKPILE_SHORT_DESC
 Transfers PP into the imperial stockpile
 
 PROJECT_BT_STOCKPILE_DESC
-'''Transfers PP at the planet where this project is enqueued to the imperial stockpile. Anything enqueued below the transfer project at a supply-connected planet will not be allocated PP.
+'''Transfers PP at the planet where this project is enqueued to the imperial stockpile. The amount to transfer and how often the transfer should occur can be specified. PP spent on the transfer are added to the imperial stockpile.
 
-Only transfers PP that are available at this planet; PP allocation on planets that are not supply-connected to here will not be affected by this project.
+Only uses PP that are produced in the same supply-group. No PP from the imperial stockpile are used.
 
-Surplus PP (PP not allocated to anything on the queue) at this or any other location will be transferred to the stockpile with or without a stockpile transfer project on the queue.  This project is used to transfer PP to the stockpile even if there are things on the queue below it that could have been allocated available PP.'''
+Surplus PP (PP not allocated to anything on the queue) at this or any other location will be transferred to the stockpile with or without a stockpile transfer project on the queue. PP used for stockpile transfer are not available for items on the build queue below it.
+
+For example: An empire wants to draw PP from the stockpile in a secondary supply-group. For this the stockpile needs to be filled from the primary supply-group. The empire has 100 PP industry in the primary supply-group and its stockpile use limit is 10 PP. To make sure that 10 PP are always available in the stockpile, one adds to the top of build queue a 10x Stockpile Transfer project on a planet in the primary supply-group.
+The Stockpile Transfer project is above other primary supply-group projects in the build queue. So it is fully funded and will increase the stockpile by 10 PP next turn. In order for a lasting effect one sets the repetition to 99 times.'''
```

Comment:
Because the ... project is above ... the queue, it is fully funded ...
To ensure a continuing transfer each turn, the repetition of the project is set to 99 times.